src/vs/workbench/api/test/browser/extHostApiCommands.test.ts

```diff
@@ -1399,6 +1401,37 @@ suite('ExtHostLanguageFeatureCommands', function () {
 		assert.ok(value[0].parent);
 	});
 
+	test('Selection Range should respect settings', async function () {
+		// TODO: how to reset this config after this test to avoid polluting other tests?
+		testConfigurationService.setUserConfiguration('editor', { smartSelect: { selectLeadingAndTrailingWhitespace: false } });
```

Comment:
you can put something into the disposables array because that's invoked after the test is done

Reply:
I spent 30 minutes looking through the codebase for an example of this (disposable config) but I couldn't find anything. Are you aware of an existing example I could borrow from?